### PR TITLE
Add UX Lead recruiting article

### PR DESCRIPTION
**Pull request by marteki**
Preview: http://marteki.github.io/cfpb.github.io/articles/we're-looking-for-an-experienced-manager-to-lead-our-UX-team/

### CODE REVIEW

**Comment by marteki** [comment]
Tagging @Scotchester @benguhin @KimberlyMunoz @ascott1

File: _posts/2015-09-11-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+published: true
+layout: post
+author: Benjamin Guhin
+title: We're looking for an experienced manager to lead our UX team
```
Comment (Scotchester) [inline _posts/2015-09-11-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Please replace apostrophe with the curly kind.

File: _posts/2015-09-11-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+published: true
+layout: post
+author: Benjamin Guhin
+title: We're looking for an experienced manager to lead our UX team
+description: "We're hiring an experienced manager to lead the disciplines of interaction design, user research, and content strategy."
```
Comment (Scotchester) [inline _posts/2015-09-11-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Please replace apostrophe and quotes with the curly kind.

File: _posts/2015-09-11-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md (hunk 1 above)
Comment (marteki) [inline _posts/2015-09-11-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Done!

File: _posts/2015-09-11-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md (hunk 2 above)
Comment (marteki) [inline _posts/2015-09-11-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Removed the unnecessary double quotes, and fixed the single quote.

**Comment by ascott1** [comment]
This is great! :computer: :muscle: :us:

**Comment by marteki** [comment]
Any hope of this being published today, or tomorrow morning at the latest? We're about to run into some other publication deadlines, and the front office would like this out soon.

**Comment by Scotchester** [comment]
Yes, I'll finish reviewing today.

**Comment by ascott1** [comment]
If it's not clear from my comment above, this gets a +1 from me. I've read this over a few times. @Scotchester may pick up on things that my quick reading brain misses though.

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+---
+
+>“When I first applied for the loan modification, I wasn’t behind, so they told me that my chances of getting a loan modification weren’t good, that they needed to see me having some difficulty, maybe being behind a few months, showing that I was having difficulty maintaining my mortgage. Well I did that, ended up getting three months behind and that’s when the notice of foreclosure letters started to come. I just became very disappointed in the whole process… I wish there was more information about how it works.”
+
+We heard this during a usability test in Memphis, Tennessee last December. I work as a User Experience (UX) Designer for the Consumer Financial Protection Bureau (CFPB), and part of our mandate as a federal agency is to accept complaints and other feedback from the public about financial products and services. My team was in Memphis testing a new prototype for our online consumer complaint form. While we received positive feedback on our design work, our participants’ ongoing struggles made it incredibly clear that we had a lot more work to do.
```
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Comma after "Tennessee".

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+I joined the Bureau in 2013 as part of our first class of [Technology and Innovation Fellows](http://www.consumerfinance.gov/jobs/technology-innovation-fellows/), which is a program that our early technology team designed in order to take advantage of design and technology talent outside of Washington, DC. Fellows serve two-year terms from anywhere in the country with occasional travel to our headquarters in DC. Our first class of 30 designers and developers quadrupled the size of the existing Design & Development Team and radically changed our ability to serve the Bureau. We continued to grow throughout 2013 and 2014, including the hiring of our [2015 class of fellows](https://cfpb.github.io/articles/welcome-2015-technology-innovation-fellows/), who started in January.
```
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: "D.C." (twice)

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+
+## The role of UX specialists in a government agency
+
+As the larger design and development group has grown, our group of UX designers has been working on the best ways to integrate user research, content strategy, and user-centered design into the Bureau’s initiatives for rulemaking, supervision, enforcement, and consumer engagement. 
+
+When I arrived two and a half years ago, employees across the organization already had a basic understanding of and appreciation for user-centered design, prototyping, and usability testing, so stakeholders and subject matter experts were already interested in participating in the design process. We didn’t need to “sell” the concept of user-centered design; our role has been to take it to the next level.
+
+Leisa Reichelt, Director of User Research for the UK’s [Government Digital Service](https://gds.blog.gov.uk/), wrote about the importance of this shared ownership of user experience in her article “[There is no UX, there is only UX](http://www.disambiguity.com/there-is-no-ux/)":
```
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Closing quotation marks should be curly.

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+>It is really important that no one in the team can point to someone over in the corner and put all the burden of user experience on that guy. No one person, no small group of people can be made responsible for the user experience of a service. It is down to the entire team to achieve this, and we need to drag people into the team who make decisions way before we get on the scene.
+
+Our UX designers, researchers, and content strategists add to this base understanding of UX through many means. We facilitate working sessions, wireframe designs, diagram complex systems, communicate functional specifications, plan user research, and work to maintain consistency across our products, platforms, and breakpoints. Jared Spool, founder of [UIE](http://www.uie.com/), recently wrote about this balance in “[Hiring UX Experts Versus Giving Your Team Their Own UX Skills](https://medium.com/uie-brain-sparks/hiring-ux-experts-versus-giving-your-team-their-own-ux-skills-c1fd9e4e480)”:
```
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Sentence case the title of Jared's article.

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+>UX is a complex field because there are lots of complexities: accessibility, mobile design, working in government agencies, working in an agile development process, and dealing with cross-cultural needs, to name a few. Yet, these are edge cases. 80% of UX work is quite routine and when learned by everyone on the team, creates consistently good designs.
+
+What this means for our UX designers is:
+
+1.	We get to work alongside economists, attorneys, data analysts, and other experts in their fields to create tools, services, and regulations to help American consumers, and
```
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Should the final comma here be a semicolon?

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+2.	We’re more able to focus on the complexity of designing for the entire American public in a marketplace known for fine print, confusing terms, and deceptive marketing.
+
+
+## What we’re doing 
+
+In four years, the CFPB has proposed new rules for mortgage servicing, prepaid cards, payday loans, money transfers, and debt collection. We’ve established partnerships with schools, libraries, and financial counselors to help educate and empower consumers. We’ve helped return over $10 billion to the American public as a result of our enforcement actions. As our team of designers and developers has grown, we’ve also launched several products and services to help advance the mission of the Bureau:
+
+* We’ve created print and online resources to help consumers [understand the home-buying process](http://www.consumerfinance.gov/owning-a-home/), [shop for a mortgage](http://www.consumerfinance.gov/owning-a-home/check-rates), and [understand the terms](http://www.consumerfinance.gov/owning-a-home/loan-options) before they sign.
```
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: The "shop for a mortgage" link is now located at http://www.consumerfinance.gov/owning-a-home/explore-rates/

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+* We’ve launched tools to help students [compare college costs](http://www.consumerfinance.gov/paying-for-college/compare-financial-aid-and-college-cost/) and [determine their repayment options](http://cfpb.github.io/articles/repay-student-debt-feature-revamp/).
```
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: https on the second link, please.

Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: (and throughout the article on cfpb.github.io links)

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+* We’ve designed [standardized disclosures](http://cfpb.github.io/articles/designing-prepaid-card-disclosures/) for prepaid cards to help consumers understand differences in costs and fees.
```
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Add "proposed" before "standardized disclosures".

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+* We’re consolidating our consumer complaint forms into a unified experience with support for complex complaints that involve multiple companies.
+* We’re developing applications to search and analyze data from consumer complaints, comments, and inquiries.
+* We’re creating new types of interactive disclosures to help ensure that consumers know all the facts when they’re making financial decisions.
+
+
+## How we work and what we’re looking for
+
+Our Design & Development Team manages this work by empowering small, cross-disciplinary teams to collaborate remotely and get things done. Our UX designers work alongside graphic designers, front- and back-end developers, data specialists, and subject matter experts every day, with some of them serving as the Scrum Master or Product Owner for their project team. We meet as cross-project “feedback houses” to help spread information among designers, each project team presents their recent work on Friday afternoons, and we use our publicly-available [Design Manual](https://cfpb.github.io/design-manual/) to maintain design and brand patterns across projects.
```
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: If we're not capitalizing the other roles, I think we should leave "scrum master" and "product owner" uncapitalized.

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+
+The next phase in our growth is hiring an experienced manager to lead our disciplines of interaction design, user research, and content strategy as a mentor, supervisor, technical expert, and visionary advocate for user-centered design. The UX Lead position will have an option for working remotely outside of Washington, DC, though it will require a significant amount of travel to our headquarters as well as other locations where we conduct user research. We want to find the best possible candidate to help our team grow, so we’re conducting a national search.
```
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: D.C.

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+
+We hear from hundreds of American consumers every day who believe that we may be able to help them find a solution to problems that are crippling their day-to-day lives. Designers are a part of the solution here. Please join us.
+
+
+
+To learn more about the CFPB’s UX Lead position, email us at [jobs@cfpb.gov](jobs@cfpb.gov).
```
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Needs to be a `mailto:` link. Also, should we add a prefilled subject string to this link?

**Comment by Scotchester** [comment]
Review complete!

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md (hunk 13 above)
Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Also, any reason we shouldn't be using `tech@cfpb.gov` here?

Comment (marteki) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: I don't know enough about the difference between the two to answer that. Who does? @benguhin?

Comment (marteki) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Also, the prefilled subject string will be "Interested in UX Lead position shown on cfpb.github.io".

Comment (Scotchester) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Well, Bruce monitors tech@ and is more likely to respond to people in a timely fashion. jobs@ at goes to OHC.

Comment (marteki) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: That's a good enough reason for me. Changing to tech@.

File: _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md (hunk 3 above)
Comment (marteki) [inline _posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md]: Fixed!

**Comment by marteki** [comment]
Think I got all the edits, @Scotchester. Thank you again! :smiley: And let me know if I missed anything.

**Comment by marteki** [comment]
Aiming for publication tomorrow morning.

**Comment by ascott1** [comment]
PUSH THE BUTTON

**Pull request merged**